packages/discovery-provider/ddl/functions/get_user_score.sql

```diff
@@ -1,13 +1,15 @@
 -- identical to get_user_scores but for a single user
 -- used to generate a user score for attestations and in UI tool
+drop function if exists get_user_score(integer);
```

Comment:
you shouldn't need this drop - it should replace it right?

Reply:
unf `create or replace` cares about the return type as well